apps/etherpad/templates/services/app/dc.yml.j2

```diff
@@ -0,0 +1,67 @@
+apiVersion: v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: etherpad
+    service: etherpad
+    version: "{{ etherpad_image_tag }}"
+    deployment_stamp: "{{ deployment_stamp }}"
+  name: "etherpad-app-{{ deployment_stamp }}"
+  namespace: "{{ project_name }}"
+spec:
+  replicas: {{ etherpad_app_replicas }}
+  template:
+    metadata:
+      labels:
+        app: etherpad
+        service: etherpad
+        version: "{{ etherpad_image_tag }}"
+        deploymentconfig: "etherpad-app-{{ deployment_stamp }}"
+        deployment_stamp: "{{ deployment_stamp }}"
+    spec:
+      # Prefer running pods on different nodes for redundancy
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                    - key: deploymentconfig
+                      operator: In
+                      values:
+                        - "etherpad-app-{{ deployment_stamp }}"
+                topologyKey: kubernetes.io/hostname
+      containers:
+        - name: etherpad
+          image: "{{ etherpad_image_name }}:{{ etherpad_image_tag }}"
+          imagePullPolicy: Always
+          command:
+            - "bash"
+            - "-c"
+            - node node_modules/ep_etherpad-lite/node/server.js --sessionkey private/SESSIONKEY.txt --apikey private/APIKEY.txt
+          livenessProbe:
+            httpGet:
+              path: /
+              port: {{ etherpad_application_port }}
+            initialDelaySeconds: 30
+            periodSeconds: 10
+          readinessProbe:
+            httpGet:
+              path: /
```

Comment:
I have no better idea at the time without modifying the application itself. Maybe we should consider integrating [terminus](https://github.com/godaddy/terminus) or [lightship](https://github.com/gajus/lightship) to add health check views to the express server but I am not sure if this can be achieved _via_ plugins development or just server script hacking.

Reply:
It looks like there is a server hook for that: https://etherpad.org/doc/v1.6.2/#index_expresscreateserver but we must create the plugin that integrates lightship.

Reply:
As this require some work to add a plugin to the image, if you agree, I would like to fix this in a separate PR. WDYT?

Reply:
So open an issue?

Reply:
I was waiting for @lunika answer :sweat_smile: Done see #489